### PR TITLE
Nextjs upgrade navbar

### PR DESCRIPTION
**Pull request by hyomiTheTech**
## Related Issue

- Information about the related issue

Closes: # 
Part of issue#164

### Describe the changes you've made

Upgrading navbar reflecting the new Figma design

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

I tested and got reviewed on the local host.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/web3community/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/WebXDAO/WebXDAO.github.io/pull/180"></a>

### CODE REVIEW

File: components/Global/Navbar.jsx [hunk 1]
```diff
@@ -1,90 +1,115 @@
...
-              onClick={() => setToggleMenu(false)}
+    <nav className="sticky w-full z-30 top-0 bg-white flex text-gray justify-between items-center">
+      <div className="pl-4 my-2 flex items-center cursor-pointer">
+        <Link
+          className="no-underline hover:no-underline font-bold text-2xl lg:text-2xl"
+          href="/"
+          passHref
+        >
+          <a>
+            <Image
```
Comment (mkubdev) [inline components/Global/Navbar.jsx:42]: Hi, we should'nt use nextjs/image due to SSR optimization. It'll break gh-pages deploy... Please use `` instead

**Review by mkubdev — CHANGES_REQUESTED** [review]
Hi @hyomiTheTech, please replace nextjs/image and it's good ✨

**Review by mkubdev — APPROVED** [review]
LGTM 🚀

**Comment by hyomiTheTech** [comment]
> LGTM 🚀

Thank you!

**Review by vinzvinci — COMMENTED** [review]
Nice start!

**Comment by hyomiTheTech** [comment]
> Nice start!

Thanks!!

**Pull request merged**